src/Spectre.Console.Extensions.Logging/SpectreConsoleLoggingExtensions.cs

```diff
@@ -1,12 +1,11 @@
 using System;
-using Microsoft.Extensions.Logging;
 using Spectre.Console.Extensions.Logging;
 
 namespace Microsoft.Extensions.Logging
 {
     public static class SpectreConsoleLoggingExtensions
     {
-        public static ILoggingBuilder AddSpectreConsole(this ILoggingBuilder loggingBuilder, SpectreConsoleLoggerConfiguration config = null)
+        private static ILoggingBuilder AddSpectreConsole(this ILoggingBuilder loggingBuilder, SpectreConsoleLoggerConfiguration config = null)
```

Comment:
Why make this private? That prevents consuming applications from using the extension method.